docs/CaliptraReleaseChecklist.md

```diff
@@ -0,0 +1,52 @@
+![OCP Logo](./images/OCP_logo.png)
+
+<p style="text-align: center;">Caliptra Release Checklist</p>
+
+<p style="text-align: center;">Version 1.1</p>
+
+<div style="page-break-after: always"></div>
+
+# Overview
+
+This document provides the signoff checklist that is used when finalizing any Caliptra release version.
+
+# Release Creation
+
+## Versioning
+
+Caliptra RTL releases may be created for new major, minor, or patch versions, as described in the [semantic versioning specification](https://semver.org/spec/v2.0.0.html). The version number is reflected in the register CPTRA_HW_REV_ID. Steps described in this document are followed for each of these releases.
```

Comment:
`reflected in the register CPTRA_HW_REV_ID`-->`reflected in the CPTRA_HW_REV_ID register`